src/components/LHNOptionsList/OptionRowLHNData.tsx

```diff
@@ -63,46 +16,48 @@ const defaultProps = {
  * re-render if the data really changed.
  */
 function OptionRowLHNData({
-    isFocused,
-    fullReport,
+    isFocused = false,
+    fullReport = null,
     reportActions,
-    personalDetails,
-    preferredLocale,
+    personalDetails = {},
+    preferredLocale = CONST.LOCALES.DEFAULT,
     comment,
-    policy,
+    policy = null,
     receiptTransactions,
-    parentReportAction,
-    transaction,
+    parentReportAction = null,
+    transaction = null,
     ...propsToForward
-}) {
+}: OptionRowLHNDataProps) {
     const reportID = propsToForward.reportID;
 
-    const optionItemRef = useRef();
+    const optionItemRef = useRef<OptionData | undefined>();
```

Comment:
This approach is more popular across the project. 
```suggestion
    const optionItemRef = useRef<OptionData | null>(null);
```

Reply:
In that case its better to stick with `undefined` there are lot of dependencies of that which expect that it will be `OptionData | undefined`